src/backend/src/appjl.jl

```diff
@@ -39,22 +39,29 @@ end
 
 ### APP ###
 
+
+## State (all constant)
+
 include("$(@__DIR__)/scenarios.jl")
 
-zones = GeoJSON.parsefile("$(@__DIR__)/../data/geometry/zones.geojson")
+const zones = GeoJSON.parsefile("$(@__DIR__)/../data/geometry/zones.geojson")
 
-zone_centroids = Array{Array{Float64,1},1}(undef,length(zones.features))
+const zone_centroids = Array{Array{Float64,1},1}(undef,length(zones.features))
 for f in zones.features
     zone_centroids[f.properties["fid"]] = Turf.centroid(f.geometry).coordinates
 end
 
 const NUM_ZONES = zone_centroids |> length
 
 # This should probably be reloaded periodically so server doesn't need to be restarted?
-links, mats, metadata = load_scenarios(packdir)
+const links, mats, metadata = load_scenarios(packdir)
```

Comment:
If we're going to have these update in-place then they can't be consts, right? Or at least it will mean we have to think more, e.g. resizing arrays and reassigning element-wise rather than a bog standard =.

Reply:
I expect all of this to go, anyway, so not bothered :)